waterbutler/providers/dropbox/provider.py

```diff
@@ -302,7 +305,9 @@ async def _contiguous_upload(self,
         :return: A dictionary of the metadata about the file just uploaded
         """
 
-        path_arg = {"path": path.full_path}
+        path_full_path = path.full_path
```

Comment:
nit: rename `path_full_path` to just `full_path`
```suggestion
        full_path = path.full_path
```